packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -23,6 +26,20 @@ export default abstract class BaseAction {
 		this.translationService = new VSTranslationService();
 	}
 
+    private logToOutputExecutionDetails(): void {
+        const vsConsoleLogger = new VSConsoleLogger()
+        if (this.executionPath) {
+            const cliConfigurationService = new CLIConfigurationService();
+            cliConfigurationService.initialize(this.executionPath);
+            cliConfigurationService.getProjectFolder(this.commandName);
+    
+            const projectInfoService = new ProjectInfoService(cliConfigurationService.getProjectFolder(this.commandName));
+            vsConsoleLogger.info(getTimestamp() + " - " + projectInfoService.getProjectName());
```

Comment:
It's a very nice work, but I think it will be easier for the user to just see the name last directory of the execution path because it will be easier to identify the project from the tree view.
I know is not well specified in the story, so maybe we can ask about it.

Reply:
During the stand up we resolved to go for the same approach as in Webstorm, so using the folder name where the project is placed.

Reply:
This makes sense. I updated the pull request to use the same approach as WebStorm.